src/install/browserFetcher.ts

```diff
@@ -95,6 +96,20 @@ function getDownloadUrl(browserName: BrowserName, revision: number, platform: Br
       ['win64', '%s/builds/webkit/%s/webkit-win64.zip'],
     ]).get(platform);
   }
+
+  if (browserName === 'ffmpeg') {
```

Comment:
I think we now have to compress our "downloading foo..." spam into a single "downloading..." item.